fetch-key-vault-secrets/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Fetch Key Vault Secrets
+
+description: Action to Fetch Secrets stored in Azure Key Vault
+
+inputs:
+  keyvault:
+    description: Name of the Key Vault the secrets are being stored
+    required: true
+
+  secrets:
+    description: Comma-separated list of secret names to fetch values from
+    required: true
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Fetch Key Vault Secrets
+      shell: bash
+      run: |
+        IFS=',' read -ra SECRET_NAMES <<< "${{ inputs.secrets }}"
+
+        for SECRET_NAME in "${SECRET_NAMES[@]}"; do
+          SECRET_VALUE=$(az keyvault secret show --name "$SECRET_NAME" --vault-name "${{ inputs.keyvault }}" --query "value" -o tsv)
+
+          echo "::add-mask::$SECRET_VALUE"
+
+          echo "SECRET_$SECRET_NAME<<EOF" >> $GITHUB_OUTPUT
```

Comment:
I'm not sure about prefixing the output with `SECRET_` - it means the behaviour of this action is different to the deprecated Azure one, but also I'm not sure if it beneficial?